jeto/static/js/controllers/domains.js

```diff
@@ -17,7 +17,7 @@ function DomainsController($scope, $routeParams, Domains, $http, createDialog, H
 
     };
     $scope.get_keys = function() {
-        $scope.domainInfo.ssl_key = null;
+        // $scope.domainInfo.ssl_key = null;
```

Comment:
Removing this breaks when using multiple domain controllers.
Let's say dc1 was selected and key1 was selected, if you select dc2 the key "key1" will still be selected in the UI, even though it's not present in the DC2.